src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -0,0 +1,291 @@
+package hex.genmodel.easy;
```

Comment:
Firstly, answer to the question why there's no model source code in test resources: 
The purpose of this java project is to wrap jar file with model downloaded from H2O server and tests have to reflect this objective. That's why we have to provide jar file, not a source code of a model. And this jar file has to be built from java file generated by H2O server - not the one written by hand.

So, in order to update test model to the one compatible with 3.6.0.11 version of h2o-genmodel library, please generate model POJO using your H2O server and create jar file based on it (I assume you are using H2O server that generates models based on 3.6.0.11 version of a library - in case of any doubts it can be checked by downloading library from http://<h2o-server-host>/3/h2o-genmodel.jar).